SimGeneral/MixingModule/python/SiPixelSimParameters_cfi.py

```diff
@@ -103,6 +104,10 @@ def _modifyPixelDigitizerForPhase1Pixel( digitizer ) :
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
 phase1Pixel.toModify( SiPixelSimBlock, func=_modifyPixelDigitizerForPhase1Pixel )
 
+# To use the label for 2018 data, uncomment next two lines, new GT needed
+#from Configuration.Eras.Modifier_run2_SiPixel_2018_cff import run2_SiPixel_2018
+#run2_SiPixel_2018.toModify(SiPixelSimBlock, SiPixelQualityLabel = 'forDigitizer',)
+
 from Configuration.ProcessModifiers.premix_stage1_cff import premix_stage1
 premix_stage1.toModify(SiPixelSimBlock,
```

Comment:
I see the two lines are already uncommented. So the feature is switched on, in contrast what is written in the PR description?
Likely the reason why the AddOn tests fail, given the error messages there.

Reply:
Hi @Martin-Grunewald, those two lines are still commented   
# To use the label for 2018 data, uncomment next two lines, new GT needed
#from Configuration.Eras.Modifier_run2_SiPixel_2018_cff import run2_SiPixel_2018
#run2_SiPixel_2018.toModify(SiPixelSimBlock, SiPixelQualityLabel = 'forDigitizer',)

Reply:
Well, it is switched on somewhere:
```
----- Begin Fatal Exception 08-Feb-2019 13:14:16 CET-----------------------
An exception of category 'NoProxyException' occurred while
   [0] Processing  stream begin Run run: 1 stream: 2
   [1] Calling method for module EventSetupRecordDataGetter/'hltGetConditions'
   [2] Using EventSetup component SiPixelQualityESProducer/'siPixelQualityESProducer' to make data SiPixelQuality/'forDigitizer' in record SiPixelQualityRcd
Exception Message:
No data of type "SiPixelQuality" with label "forDigitizer" in record "SiPixelQualityFromDbRcd"
 Please add an ESSource or ESProducer to your job which can deliver this data.
----- End Fatal Exception -------------------------------------------------
```